views/layouts/showlite.handlebars

```diff
@@ -9,16 +9,16 @@
     <link rel="stylesheet" href="/assets/css/general.css" type="text/css">
     <link rel="stylesheet" href="/assets/css/mediaQueries.css" type="text/css">
 	<meta property="fb:app_id" content="1371961932852223">
-	{{#unless fileInfo.nsfw}}
```

Comment:
Is it possible for `showlite` and `show` (and possibly other layouts) to share more of their common logic (particularly `<head>`), inside of a shared template?

Reply:
There may be more room for synergy, but for now I condensed the common header tags into a handlebars helper and tightened up the open graph/twitter tag helper.